test/exec-tests.ts

```diff
@@ -367,18 +367,33 @@ describe('Execution tests', async () => {
             });
 
             expect(options).toEqual({});
-            expect(args).toEqual([
-                '--config',
-                'etc/nsjail/sandbox.cfg',
-                '--cwd',
-                '/app',
-                '--bindmount',
-                '/tmp/hellow:/app',
-                '--env=LD_LIBRARY_PATH=/usr/lib:/app/lib',
-                '--env=HOME=/app',
-                '--',
-                './output.s',
-            ]);
+            if (process.platform === 'win32') {
+                expect(args).toEqual([
+                    '--config',
+                    'etc/nsjail/sandbox.cfg',
+                    '--cwd',
+                    '/app',
+                    '--bindmount',
+                    '/tmp/hellow:/app',
+                    '--env=LD_LIBRARY_PATH=/usr/lib;/app/lib',
```

Comment:
took me a while to spot the different here; for my poor old eye maybe:
```ts
const path_sep = windows ? ':' : ';';
expect(args).toEqual(...
'--env=LD_LIBRARY_PATH=/usr/lib:${path_sep}/app/lib
```

or something?